### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by edwardbaeg**
@DavidPatterson-Cole, I should have called this branch setup/postgres. Let me know what you think!

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS photos;
```
Comment (DavidPatterson-Cole) [inline database/schema.sql:1]: Why not just drop and create database

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+DROP TABLE IF EXISTS restaurants;
+
+CREATE TABLE restaurants (
+  id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
```
Comment (DavidPatterson-Cole) [inline database/schema.sql:5]: Does this auto start at 1 not 0?

Comment (DavidPatterson-Cole) [reply database/schema.sql:5]: If you have a lot of data you will get better speed by defining as not null and then altering to primary key later

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  restaurantName VARCHAR (50) NOT NULL,
+  username VARCHAR (50) NOT NULL,
+  description VARCHAR (500) NOT NULL,
+  createdAt VARCHAR (50) NOT NULL,
+  photoUrls VARCHAR (500)
+);
+
+\COPY restaurants FROM 'combined.tsv' DELIMITER E'\t';
```
Comment (DavidPatterson-Cole) [inline database/schema.sql:13]: What is the E for?

**Review by DavidPatterson-Cole — COMMENTED** [review]
Nice work - a few comments/questions

File: database/schema.sql (hunk 1 above)
Comment (edwardbaeg) [reply database/schema.sql:1]: I didn't find that to be necessary.

File: database/schema.sql (hunk 2 above)
Comment (edwardbaeg) [reply database/schema.sql:5]: Thanks for the heads up!

File: database/schema.sql (hunk 3 above)
Comment (edwardbaeg) [reply database/schema.sql:13]: The `E` is necessary for delimiters that are over 1 character

**Pull request merged**